brainscore_core/submission/endpoints.py

```diff
@@ -0,0 +1,210 @@
+"""
+Process plugin submissions (data, metrics, benchmarks, models) and score models on benchmarks.
+"""
+import traceback
+import logging
+from abc import ABC
+from datetime import datetime
+import http
+import os
+import random
+import requests
+from requests.auth import HTTPBasicAuth
+import string
+from typing import List, Union, Dict
+
+from brainscore_core import Benchmark, Score
+from brainscore_core.submission import database_models
+from brainscore_core.submission.database import connect_db, modelentry_from_model, \
+    submissionentry_from_meta, benchmarkinstance_from_benchmark, update_score, \
+    public_model_identifiers, public_benchmark_identifiers, uid_from_email, email_from_uid
+
+logger = logging.getLogger(__name__)
+
+
+def process_github_submission(plugin_info: Dict[str, Union[List[str], str]]):
+    """
+    Triggered when changes are merged to the GitHub repository, if those changes affect benchmarks or models.
+    Starts parallel runs to score models on benchmarks (`run_scoring`).
+    """
+    jenkins_base = "http://braintree.mit.edu:8080"
+    jenkins_usr = os.environ['JENKINS_USR']
```

Comment:
why don't we like the second `e`?

Reply:
OK, I'm pulling an additional `e` out of the budget just for you.

Reply:
```suggestion
    jenkins_user = os.environ['JENKINS_USER']
```